### PR TITLE
fix bug list token scroll

### PR DESCRIPTION
**Pull request by nguyenhoaidanh**
- update text not found token in bridge
- fix bug sroll token list
image

### CODE REVIEW

File: src/components/SearchModal/bridge/CurrencyListBridge.tsx [hunk 1]
```diff
@@ -73,7 +73,7 @@ const CurrencyListBridge = memo(function CurrencyListV2({
   )
 
   return (
-    <div style={{ flex: '1', overflow: 'auto', height: '100%' }}>
+    <div style={{ flex: '1', overflow: 'hidden', height: '100%' }}>
```
Comment (viet-nv) [inline src/components/SearchModal/bridge/CurrencyListBridge.tsx:76]: how about scrollY?

Comment (nguyenhoaidanh) [reply src/components/SearchModal/bridge/CurrencyListBridge.tsx:76]: actually. it's not scrollable div, scrollable is children of this div.

Comment (nguyenhoaidanh) [reply src/components/SearchModal/bridge/CurrencyListBridge.tsx:76]: tested on firefox, safari, chrome and brave. Trust me :D

Comment (viet-nv) [reply src/components/SearchModal/bridge/CurrencyListBridge.tsx:76]: okay trusted you for this time :v

**Review by viet-nv — APPROVED** [review]

**Pull request merged**